src/js/elements/subindicator_filter/filter_controller.js

```diff
@@ -106,9 +106,13 @@ export class FilterController extends Component {
         })
     }
 
-    setFilterVisibility(){
-        if (this.model.dataFilterModel.availableFilters.length <= 0) {
+    setFilterVisibility() {
+        let nonAggregatableGroups = this.model.dataFilterModel.nonAggregatableGroups;
```

Comment:
why let and not const?

I think you should use const by default and only use let if you need to be able to assign a value after declaration.